browser-test/utils.js

```diff
@@ -56,3 +53,5 @@ export const urls = {
   login: `http://0.0.0.0:8080/${orgSlug}/login`,
   status: `http://0.0.0.0:8080/${orgSlug}/status`,
 };
+
+jest.setTimeout(10000);
```

Comment:
I increased the global jest time out here.

Reply:
Yes, it needs to be increased.